tamr_toolbox/data_io/dataframe.py

```diff
@@ -372,7 +404,14 @@ def validate(
         _check_nonnull_columns(df_profile, require_nonnull_columns=require_nonnull_columns).details
     )
 
-    # convert to dict for printing/return
+    if custom_check is not None:
+        for i in range(len(custom_check)):
+            failed_checks_dict.update(
+                _check_custom(
+                    df, check_function=custom_check[i][0], columns_to_check=custom_check[i][1],
+                ).details
+            )
```

Comment:
It's not very Pythonic to iterate using range, better to do
```suggestion
        for custom_check in custom_checks:
            failed_checks_dict.update(
                _check_custom(
                    df, check_function=custom_check[0], columns_to_check=custom_check[1],
                ).details
            )
```